README.md

```diff
@@ -31,15 +31,24 @@ This is a special set of tests that use the [Verify](https://github.com/VerifyTe
 
 If you've verified the changes and decided they are valid, you can accept them  using `AcceptApiChanges.ps1` or `AcceptApiChanges.sh`. Alternatively, you can use the [Verify Support](https://plugins.jetbrains.com/plugin/17240-verify-support) plug-in to compare the changes and accept them right from inside Rider. See also the [Contribution Guidelines](CONTRIBUTING.md).
 
-# Powered By
-<a href="https://aws.amazon.com/"><img src="docs/assets/images/aws.png" style="width:100px"/></a>&nbsp;
```

Comment:
AWS is also to on `docs/index.html:82`

Reply:
Yes, I know. Hence PR #2285 targeting `master`

Reply:
aha, didn't notice the target branch.
Just noticed that when running the website locally, the front page was broken because this PR deleted aws.png, but still referenced it from index.html